addons/mail/static/tests/legacy/web/activity/activity_widget_tests.js

```diff
@@ -379,6 +379,62 @@ test("list activity widget: open dropdown", async () => {
     await contains(".o-mail-ListActivity-summary", { text: "Meet FP" });
 });
 
+test("list activity widget: cancel an activity", async () => {
+    const pyEnv = await startServer();
+    const activityTypeId_1 = pyEnv["mail.activity.type"].create([{}]);
+    const activityId_1 = pyEnv["mail.activity"].create([
```

Comment:
```suggestion
    const activityTypeId = pyEnv["mail.activity.type"].create([{}]);
    const activityId = pyEnv["mail.activity"].create([
```

Reply:
Done :+1:

Reply:
Hey @khpa-odoo,

Adapt the pr description to reflect a test case created to ensure the cancel button functionality.

Reply:
Added : )